packages/core/src/login/webview/vue/login.vue

```diff
@@ -359,6 +370,37 @@ export default defineComponent({
             this.regions = regions
         },
         async emitUpdate(cause?: string) {},
+        async updateExistingConnections() {
+            const connections = await client.fetchConnections()
+            connections?.forEach((connection, index) => {
+                this.existingLogins.push({
+                    id: LoginOption.EXISTING_LOGINS + index,
+                    text: 'Used by AWS Toolkit',
+                    title: isBuilderId(connection.startUrl)
+                        ? 'AWS Builder ID'
+                        : `AWS IAM Identity Center ${connection.startUrl}`,
+                    connectionId: connection.id,
+                })
+            })
+            this.$forceUpdate()
+            // in case toolkit activation is slow.
+            // Todo:
```

Comment:
iirc there is a utility function that activates extensions (and does nothing if it is already activated). Let's call it here instead of waiting

Reply:
https://github.com/aws/aws-toolkit-vscode/blob/master/packages/core/src/shared/utilities/vsCodeUtils.ts#L125

Reply:
Done in latest commit